R/NAMESPACE

```diff
@@ -36,7 +36,8 @@ export(step_client)
 export(task_client)
 export(test)
 export(version_info)
+import(reticulate)
 importFrom(R6,R6Class)
 importFrom(magrittr,"%>%")
-importFrom(reticulate,import_builtins)
-importFrom(reticulate,py_get_attr)
+importFrom(rstudioapi,hasFun)
+importFrom(rstudioapi,restartSession)
```

Comment:
Can we avoid the dependency on `rstudioapi` for now? or place a soft dependency instead? Right now the code fails because `rstudioapi` is not listed as a metaflow dependency.